src/rust/src/conversion_r_to_s.rs

```diff
@@ -153,10 +153,11 @@ fn recursive_robjname2series_tree(x: &Robj, name: &str) -> pl::PolarsResult<Seri
         }
 
         Rtype::Raw => {
-            let rpolars_raw_list = list!(x)
+            let mut binding = list!(x);
```

Comment:
modifying attributes modifies in place. Previously, this didn't require a mutable reference giving false confidence. Now it requires a mutable reference. Additionally, the same type is returned after modifying the attributes. So if you use `set_class()` or `set_attrib()` on a `List` it will return a `mut List`.